src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -63,13 +59,17 @@
             <div
               v-bind="{[SCROLL_LOCK_DISABLE_ATTR]: true}"
               class="quick-navigation__refs"
+              @keydown.down.exact.prevent="focusNext"
+              @keydown.up.exact.prevent="focusPrev"
+              @keydown.enter.exact="handleKeyEnter"
+              @click.self="closeQuickNavigationModal"
```

Comment:
Not sure this click handler does much 🤔 

The modal already handles closing, when you click outside.

Reply:
https://github.com/apple/swift-docc-render/pull/736/commits/8d8c17e136086ff633c105f2f4aa570e54361551